app/lib/navigation/state_file_nj_question_navigation.rb

```diff
@@ -55,15 +55,15 @@ def self.show_progress?(controller_class)
                                           Navigation::NavigationStep.new(StateFile::Questions::NjTenantPropertyTaxWorksheetController), # Line 40a
                                           Navigation::NavigationStep.new(StateFile::Questions::NjTenantRentPaidController), # Line 40a
                                           # question after property taxes set in NjPropertyTaxFlowOffRamp
-                                      ]),
+                                        ]),
       Navigation::NavigationSection.new(I18n.t("state_file.navigation.nj.section_4", filing_year: MultiTenantService.statefile.current_tax_year), [
                                           Navigation::NavigationStep.new(StateFile::Questions::NjSalesUseTaxController), # Line 51
                                           Navigation::NavigationStep.new(StateFile::Questions::NjEstimatedTaxPaymentsController), # Line 57
-                                      ]),
+                                        ]),
       Navigation::NavigationSection.new("state_file.navigation.nj.section_5", [
                                           Navigation::NavigationStep.new(StateFile::Questions::PrimaryStateIdController), # Footer
                                           Navigation::NavigationStep.new(StateFile::Questions::SpouseStateIdController), # Footer
-                                          Navigation::NavigationStep.new(StateFile::Questions::NjReviewController), # review controller should come before taxes owed / refund due screens
+                                          Navigation::NavigationStep.new(StateFile::Questions::NjReviewController), # REVIEW: controller should come before taxes owed / refund due screens
```

Comment:
[dust] this change to this comment seems like it changes the meaning here in a way I wouldn't expect - makes it sound like a TODO command instead of a note. Is this intentional?

Reply:
tbh is this comment even necessary at all, in its original form